trunk/source/Library-Automata/src/de/uni_freiburg/informatik/ultimate/automata/nestedword/operations/IsEmptyHeuristic.java

```diff
@@ -83,7 +84,9 @@ public final class IsEmptyHeuristic<LETTER, STATE> extends UnaryNwaOperation<LET
 	private final Predicate<STATE> mIsForbiddenState;
 	private final NestedRun<LETTER, STATE> mAcceptingRun;
 	private final STATE mDummyEmptyStackState;
-
+	protected final List<Pair<STATE, LETTER>> mWayPoints;
+	boolean stillFollowingWayPoint = true;
+	boolean mDontVisitLoops = false;
```

Comment:
- Please indicate visibility for these fields. Is there any reason to not make them private?
- The first one also does not conform to our naming conventions.